### PR TITLE
Update CI configs

### PR DESCRIPTION
**Pull request by XhmikosR**
@mattbasta: please squash these if tests pass; I just edited through the web interface hence the multiple patches.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
+sudo: false
 language: node_js
 node_js:
   - "6"
-  - "7"
   - "8"
-sudo: false
+matrix:
+  fast_finish: true
```
Comment (mattbasta) [inline .travis.yml]: I'd prefer not to add `fast_finish`

File: appveyor.yml [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
-  - node --version
-  - npm --version
-  # run tests
+  - node --version && npm --version
   - npm test
 
 build: off
+
+matrix:
+  fast_finish: true
```
Comment (mattbasta) [inline appveyor.yml]: same

File: .travis.yml [hunk 3]
```diff
@@ -1,6 +1,10 @@
+sudo: false
 language: node_js
 node_js:
   - "6"
-  - "7"
   - "8"
```
Comment (mattbasta) [inline .travis.yml]: If you're going to update the list of supported versions, please add 9

Comment (XhmikosR) [reply .travis.yml]: Nah, I just wanted to remove the non-LTS version.

**Pull request merged**